spec/board_spec.rb

```diff
@@ -18,4 +20,46 @@
             expect(@board.cells).to be_a(Hash)
         end
     end
+
+    describe '#validating coordinates' do
+        it 'can make sure coordinates are the correct length for the ship' do
+
+            expect(@board.valid_placement?(@cruiser, ["A1", "A2"])).to eq(false)
+            expect(@board.valid_placement?(@submarine, ["A2", "A3", "A4"])).to eq(false)
+        end
```

Comment:
This looks good! I think we may want to add some additional tests to verify the ability to return true.